gulpfile.js

```diff
@@ -289,7 +354,7 @@ exports.watch = watch;
 //    which is generally how this script is always used (except in
 //    rare case where the user is running the watch task).
 if (config.install) {
-    exports.default = gulp.series(install)
+    exports.default = gulp.series(install);
 } else {
-    exports.default = gulp.series(clean, ingest, styles, sync);
+    exports.default = gulp.series(clean, ingest, styles, build_modernizr, sync);
```

Comment:
If this is building a package out, then it should prob go into the prior exports.default which is where all the packages are collected:
```
exports.default = gulp.series(install, build_modernizr);
```

Reply:
(up to you, though, since either way it is going to get built :) )

Reply:
actually, disregard my post. makes more sense for this to be in the `npm run build` like you have :)

Reply:
@bradymiller Yeah it needs to run after styles cause it's using public assets